backend.native/compiler/ir/backend.native/src/org/jetbrains/kotlin/backend/konan/serialization/KonanIr.proto

```diff
@@ -2,7 +2,7 @@
 syntax = "proto2";
 package org.jetbrains.kotlin.serialization;
 
-import "org/jetbrains/kotlin/backend/konan/serialization/KonanLinkData.proto";
+import "org/jetbrains/kotlin/backend/konan/serialization/descriptors.proto1";
```

Comment:
В KonanLinkData.proto писал:

```
// This, and all the rest of the included proto files have ".proto1" extension.
// That allows us to construct the current file,
// but protoc will skip .class files for them.
// The issue here is that we need to co-exist with the big Kotlin,
// in the same namespace of renamed org.jetbrains.kotlin.protobuf packages.
// In case we merge to the main Kotlin workspace the scheme will be simplified.
import "org/jetbrains/kotlin/backend/konan/serialization/descriptors.proto1";
```